src/metabase/driver/FixedHiveConnection.clj

```diff
@@ -0,0 +1,22 @@
+(ns metabase.driver.FixedHiveConnection
+  (:import [org.apache.hive.jdbc HiveConnection]
+           [java.sql ResultSet SQLException]
+           java.util.Properties)
+  (:gen-class
+   :extends org.apache.hive.jdbc.HiveConnection
+   :init init
+   :constructors {[String java.util.Properties] [String java.util.Properties]}))
+
+(set! *warn-on-reflection* true)
```

Comment:
we don't want this in the codebase because it will lead to tons of warning messages for things in our dependencies making it needlessly verbose for people just trying to use Metabase without doing development on it

Reply:
Done